crates/store/re_log_types/src/arrow_msg.rs

```diff
@@ -10,7 +10,9 @@ use arrow::array::RecordBatch as ArrowRecordBatch;
 use crate::TimePoint;
 
 // TODO(#3741): Remove once <https://github.com/apache/arrow-rs/issues/6803> is released
+#[cfg(feature = "serde")]
```

Comment:
What does `serde` has to do with Arrow..? :thinking:

Reply:
Nothing - just a drive-by-fix, similar to this:
* https://github.com/rerun-io/rerun/pull/8734